index.html

```diff
@@ -3,88 +3,108 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport"content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>paz</title>
     <link rel="stylesheet" type="text/css" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
     <link href="dist/main.css" rel="stylesheet">
+    <link rel="stylesheet" href="node_modules/swiper/dist/css/swiper.css">
     <script type="text/javascript" src="node_modules/jquery/dist/jquery.min.js"></script>
 </head>
-<body>
-    <div class="wrapper">
-    	
-   	<!-- HEADER 1 -->
+<body>	
+		<!-- HEADER 1 -->
+	<header class="head">
 
-    	<header class="head">
-
-    	<!-- logo and search sec-->
-
-	    	<div class="main-nav container-fluid d-flex  flex-column">
-	    		<div class="container top-sec-nav align-items d-flex justify-content-between align-items-center col">
-		    		<div class="search">
-		    			<img src="img/paz-serch-icon.png">
-		    		</div>
-		    		<div class="logo">
-			    		<img src="img/paz-logo.png">
-					</div>	
-		    		<div class="lng hidden-md-down">
-			    		ENG
-					</div>
-					<div class="nav-mobile hidden-lg-up">
-			    		<img src="img/nav-icon-mobil.png">
-					</div>
+		<!-- logo and search sec-->
+    	<div class="main-nav container-fluid d-flex  flex-column">
+    		
+    		<!-- top section -->
+    		<div class="container top-sec-nav align-items-center d-flex justify-content-between align-items-center col">
+	    		<div class="search">
+	    			<img src="img/search.svg" class="search">
+	    		</div>
+	    		<div class="logo">
+		    		<img src="img/pazlogo.png"
+	     				srcset="img/pazlogo@2x.png 2x,
+	           					img/pazlogo@3x.png 3x"
+	      				class="pazlogo">
+				</div>	
+	    		<div class="lng d-none d-lg-block">
+		    		ENG
+				</div>
+				<div class="manu-mobile d-lg-none">
+		    		<img src="img/menu.png"
+				     srcset="img/menu@2x.png 2x,
+				             img/menu@3x.png 3x"
+				     class="menu">
 				</div>
-			
-		<!-- main nav sec -->
+			</div>
+		
+			<!-- main nav sec -->
+			<div class="low-sec-nav container d-none d-lg-block">	
+	    		<nav class="nav-1 navbar navbar-nav navbar navbar-expand-lg">
```

Comment:
naming stuff `nav-1` or `button-1` is not good because it's not descriptive. Having descriptive classnames is _very_ important to an organized project. You should change this wherever you named stuff `something-x`.
Examples: 

- `nav-1` to `main-navigation` 
- `nav-2` to `pazomat-header`
- `button-1` to `pazomat-signup` 
- `button-2` to `pazomat-login` 
- `button-3` to `pazomat-mobile-menu-trigger`

This way you know what everything is by just looking at the classname